### DISCUSSION TITLE
Is possible run WireGuard server on Talos Linux?

### DISCUSSION THREAD

**Discussion by mikucat0309** [siderolabs/talos#7839] [Q&A] [answered]
I want to run a WireGuard server on Talos, but it has some [limitations](https://www.talos.dev/v1.5/learn-more/process-capabilities/), so is this possible?

**Comment by Preisschild** [comment, marked as answer]
Sure, you only net `CAP_NET_ADMIN` for wireguard.

Talos enables the kernel module by default, so you don't need `CAP_SYS_MODULE` to load the module inside a container.

https://github.com/siderolabs/pkgs/blob/main/kernel/build/config-amd64#L2317

**Reply by mikucat0309** [reply]
Thank you very much!
